### PR TITLE
react-mui-shared: update to storybook v7 (GEA-12488)

### PR DESCRIPTION
**Pull request by kenany**

### CODE REVIEW

**Review by pangea-andrest — DISMISSED** [review]

**Comment by cj-f** [comment]
Noticing this error trying to view FieldsFieldsPreview 

```
The requested module '/src/components/PangeaFields/FieldsPreview/index.tsx' does not provide an export named 'FieldsPreviewProps'
SyntaxError: The requested module '/src/components/PangeaFields/FieldsPreview/index.tsx' does not provide an export named 'FieldsPreviewProps'
```

**Comment by kenany** [comment]
> Noticing this error trying to view FieldsFieldsPreview
Fixed.

File: packages/react-mui-shared/src/components/JsonViewer/index.tsx [hunk 1]
```diff
@@ -1,13 +1,8 @@
-import { FC, lazy } from "react";
+import { FC } from "react";
 import { Box } from "@mui/material";
 import { useTheme } from "@mui/material/styles";
 import { useReactJsonViewHighlight, Highlight } from "./utils";
-import { ReactJsonViewProps } from "react-json-view";
-
-let ReactJson: FC<ReactJsonViewProps> = () => null;
-if (typeof window !== "undefined") {
-  ReactJson = require("react-json-view").default;
```
Comment (kenany) [inline packages/react-mui-shared/src/components/JsonViewer/index.tsx:9]: @cj-f Do you remember why this `require()` is/was needed?

Comment (cj-f) [reply packages/react-mui-shared/src/components/JsonViewer/index.tsx:9]: Somewhat, I don't remember the exact error but essentially the package could only be used within the browser and trying to use the package within nextjs was causing issues, which I believe I got around by dynamically loading the package although that was a bad end-user experience, which where I then came across something mentioning to do this

Comment (kenany) [reply packages/react-mui-shared/src/components/JsonViewer/index.tsx:9]: I just threw this as-is into our pangea-integration-nextjs-authn example and it renders fine, so I think we're good to go?

Comment (cj-f) [reply packages/react-mui-shared/src/components/JsonViewer/index.tsx:9]: Ok, lets try it

**Pull request merged**